components/ingest-service/pipeline/chef_run.go

```diff
@@ -22,14 +22,15 @@ type ChefRunPipeline struct {
 
 // NewChefRunPipeline Create a new chef run pipeline
 func NewChefRunPipeline(client backend.Client, authzClient iam_v2.ProjectsClient,
-	nodeMgrClient manager.NodeManagerServiceClient, maxNumberOfBundledRunMsgs int) ChefRunPipeline {
+	nodeMgrClient manager.NodeManagerServiceClient,
+	maxNumberOfBundledRunMsgs, numberOfRunMsgTransformers int) ChefRunPipeline {
 	var (
 		in            = make(chan message.ChefRun, 100)
 		counter int64 = 0
 	)
 
 	chefRunPipeline(in,
-		processor.BuildTransmogrify(9),
+		processor.BuildTransmogrify(numberOfRunMsgTransformers),
```

Comment:
we envisioned this would happen one day. that day has come!

![tenor-234221732](https://user-images.githubusercontent.com/5712253/61201417-bae78480-a6e4-11e9-8c64-ce9a35e5a7aa.gif)